packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -203,7 +204,15 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
                     items={allLocations}
                     itemToString={(location) => location?.display ?? ''}
                     label={t('location', 'Location')}
-                    onBlur={onBlur}
+                    /*
+                      TODO: ComboBox bug does not trigger onChange below @carbon/react version 1.72 (https://github.com/carbon-design-system/carbon/issues/18145#issuecomment-2521936772)
+                      onBlur shall be refactored to onBlur={onBlur}
```

Comment:
What exactly is the TODO here?

Reply:
The todo means that the original onChange method should be used if the core has a more updated version for @carbon/react which does not have the but there - but this would required bumping and testing packages in core which i thought is not that simple. But it can be left out as well if you prefer.